src/components/shared/form-components/drop-down/drop-down.jsx

```diff
@@ -1,18 +1,26 @@
-import { useState, useRef } from "react";
+import { useState, useRef, useEffect } from "react";
 import DropDownMenu from "./list/list";
 import DownArrow from "svg/down-arrow";
+import { useRouter } from "next/router";
 
 export default function DropDown({
   isReactHookForm = false,
   setValue = "",
   listItems = [],
   attributes = {},
-  classNames = '',
-  label = '',
+  classNames = "",
+  label = "",
   handleChange,
 }) {
   const [isToggled, setIsToggled] = useState(false);
   const inputRef = useRef(null);
+  const { asPath, query } = useRouter();
+
+  useEffect(() => {
+    if (query.collection) {
+      inputRef.current.value = query.collection;
```

Comment:
No changes necessary but just a question - how come you opted to use refs instead of state?

Reply:
Just went with what I know. Whenever I start messing without value on input elements it never works properly for me. Probably something I should look at.

Reply:
Gotcha. This is definitely something worth learning for the future, using state is the standard and there are tons of examples online.